code_analyzer/githomes.py

```diff
@@ -4,7 +4,7 @@
 from common import dbConn, config
 from datetime import datetime, timedelta
 import base64
-import httplib2
+import requests
```

Comment:
The reason is that httplib2 has a wonderful "feature" of not sending authentication headers unless the server responded with 401 to the request without those headers.